### ISSUE TITLE
Numerous errors in the example files

### ISSUE THREAD

**Issue by wielandb** [zauberzeug/nicegui#454]
Unsure if every problem should be its own issue, but I feared opening a new issue for every problem with an example would be too spammy, so I am collecting them here:

**examples/ai_interface:**

- [x] Attempting to run the OpenAI Whisper Part of this Demo will always throw the following error:
```
Traceback (most recent call last):
  File "C:\Python39\lib\site-packages\nicegui\background_tasks.py", line 54, in _handle_task_result
    task.result()
  File "C:\Python39\lib\site-packages\nicegui\events.py", line 275, in wait_for_result
    await result
  File "C:\Users\me\nicegui-main\nicegui-main\examples\ai_interface\main.py", line 21, in transcribe
    prediction = await io_bound(model.predict, audio=io.BytesIO(e.content))
TypeError: a bytes-like object is required, not 'SpooledTemporaryFile'
```

- [x] Attempting to run the stable diffusiuon demo will always result in the follwing error:
```
File "C:\Python39\lib\site-packages\replicate\model.py", line 14, in predict
    raise ReplicateException(
replicate.exceptions.ReplicateException: The `model.predict()` method has been removed, because it's unstable: if a new version of the model you're using is pushed and its API has changed, your code may break. Use `version.predict()` instead. See https://github.com/replicate/replicate-python#readme
```

**examples/local_file_picker:**

- [x] Attempting to open a file will always result in the error:
```
Traceback (most recent call last):
  File "C:\Python39\lib\site-packages\nicegui\background_tasks.py", line 54, in _handle_task_result
    task.result()
  File "C:\Python39\lib\site-packages\nicegui\events.py", line 275, in wait_for_result
    await result
  File "C:\Users\me\nicegui-main\nicegui-main\examples\local_file_picker\main.py", line 8, in pick_file
    result = await local_file_picker('~', multiple=True)
  File "C:\Users\me\nicegui-main\nicegui-main\examples\local_file_picker\local_file_picker.py", line 30, in __init__
    self.grid = ui.aggrid({
AttributeError: module 'nicegui.ui' has no attribute 'aggrid'
```

**examples/map:**

- [x] The embedded map is missing proper attribution. This is not a drastic problem given the nature of the project, but in case someone does indeed use this example as a starting off point for a project, it would be great to have proper attribution by default. :)

**examples/script_executor:**

- [ ] Every single button will raise a `NotImplementedError`.

**Comment by falkoschindler** [comment]
Thanks for spotting these issues, @wielandb! Although we run all examples as part of our CI setup, we only check if they are starting. Regressions that occur when actually _interacting_ with the example UIs are easily missed.

Regarding the `local_file_picker`:
On main we introduced `ui.aggrid`, which will be released in 1.1.10. So I think the example is ok.

**Comment by falkoschindler** [comment]
@rodja I think this issue has been closed automatically, although two out of five tasks are still incomplete.

**Comment by falkoschindler** [comment]
@wielandb Can you give more details and/or a stack trace about the `NotImplementedError` in the script_executor example? I can't reproduce the problem. What version of NiceGUI do you have installed - or are you on main?

**Comment by wielandb** [comment]
> @wielandb Can you give more details and/or a stack trace about the `NotImplementedError` in the script_executor example? I can't reproduce the problem. What version of NiceGUI do you have installed - or are you on main?

The full stack trace is as follows:

```
NiceGUI ready to go on http://0.0.0.0:8080
Task raised an exception
Traceback (most recent call last):
  File "C:\Python39\lib\site-packages\nicegui\background_tasks.py", line 54, in _handle_task_result
    task.result()
  File "C:\Users\me\nicegui\examples\script_executor\main.py", line 12, in run_command
    process = await asyncio.create_subprocess_exec(
  File "C:\Python39\lib\asyncio\subprocess.py", line 236, in create_subprocess_exec
    transport, protocol = await loop.subprocess_exec(
  File "C:\Python39\lib\asyncio\base_events.py", line 1661, in subprocess_exec
    transport = await self._make_subprocess_transport(
  File "C:\Python39\lib\asyncio\base_events.py", line 493, in _make_subprocess_transport
    raise NotImplementedError
NotImplementedError
```

My nicegui version is the latest main branch on Mar 7 2023. My asyncio version is 3.4.3

**Comment by rodja** [comment]
Looks like an Windows problem. This may be related: https://stackoverflow.com/questions/44633458/why-am-i-getting-notimplementederror-with-async-and-await-on-windows

**Comment by rodja** [comment]
Whats your Python version?

**Comment by wielandb** [comment]
> Whats your Python version?

Python 3.9.2

**Comment by rodja** [comment]
Do other Windows users have the same problem? It seems like a problem with `asyncio.create_subprocess_exec` and not some core NiceGUI code. @wielandb would you like to investigate this further? Maybe by trying out other examples outside of NiceGUI which use this method?

**Comment by rodja** [comment]
@wielandb I'll close this issue to properly release 1.1.10. Feel free to open another one which focusses on the `asyncio.create_subprocess_exec` problem.

**Comment by acsr** [comment]
I still get the "TypeError: a bytes-like object is required, not 'SpooledTemporaryFile'" error with 
examples/ai_interface while attempting to run the OpenAI Whisper Part of this Demo. 

I am on M1 Apple Silicon under Monterey and Python 3.11.4, nicegui==1.3.5, replicate==0.9.0

full requirements installed:
aiofiles==23.1.0
annotated-types==0.5.0
anyio==3.7.1
bidict==0.22.1
certifi==2023.7.22
charset-normalizer==3.2.0
click==8.1.6
docutils==0.20.1
fastapi==0.100.0
fastapi-socketio==0.0.10
h11==0.14.0
httpcore==0.17.3
httptools==0.6.0
httpx==0.24.1
idna==3.4
isort==5.12.0
itsdangerous==2.1.2
Jinja2==3.1.2
markdown2==2.4.10
MarkupSafe==2.1.3
nicegui==1.3.5
orjson==3.9.2
packaging==23.1
pscript==0.7.7
pydantic==2.0.3
pydantic_core==2.3.0
Pygments==2.15.1
python-dotenv==1.0.0
python-engineio==4.5.1
python-multipart==0.0.6
python-socketio==5.8.0
PyYAML==6.0.1
replicate==0.9.0
requests==2.31.0
sniffio==1.3.0
starlette==0.27.0
success==1.0.0
typing_extensions==4.7.1
urllib3==2.0.4
uvicorn==0.22.0
uvloop==0.17.0
vbuild==0.8.1
watchfiles==0.19.0
websockets==11.0.3

**Comment by rodja** [comment]
@acsr that's strange. Can you provide the full output?

**Comment by falkoschindler** [comment]
Thanks, @acsr! I fixed it in [1f15fe4](https://github.com/zauberzeug/nicegui/commit/1f15fe493b8bfaf709920dfd79cff04c7b1e568c).
